src/pkg/packaging/windows/vscomponents/generatenupkg.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param(
+    [Parameter(Mandatory=$true)][string]$MsiPath,
+    [Parameter(Mandatory=$true)][string]$MsiVersion,
+    [Parameter(Mandatory=$true)][string]$NuspecFile,
+    [Parameter(Mandatory=$true)][string]$NupkgFile,
+    [Parameter(Mandatory=$true)][string]$Architecture,
+    [Parameter(Mandatory=$true)][string]$ComponentName,
+    [Parameter(Mandatory=$true)][string]$ComponentFriendlyName,
+    [Parameter(Mandatory=$true)][string]$ProjectUrl,
+	[Parameter(Mandatory=$true)][string]$BinDir
+)
+
+$NuGetExe = Join-Path $BinDir "nuget.exe"
+$OutputDirectory = [System.IO.Path]::GetDirectoryName($NupkgFile)
+
+if (-not (Test-Path $NuGetDir)) {
```

Comment:
Looks like this ref to `NuGetDir` wasn't cleaned up in the change to `BinDir`. I queued a new build with this if block removed: https://dev.azure.com/dnceng/internal/_build/results?buildId=237460

Reply:
Fixed.